scripts/ci/docker_upgrade.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+set -e
+
+check_parameters() {
+  echo "-- check_parameters"
+
+  if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${DOCKER_REPOSITORY}" ]; then
+    echo "\${IMAGE_NAME}, \${IMAGE_VARIANT} and \${DOCKER_REPOSITORY} need to be defined."
+
+    false
+  fi
```

Comment:
This would also be a bit nicer with a helper:

```suggestion
  [[ $IMAGE_NAME != "" ]] || die '${IMAGE_NAME} must be defined.'
  [[ $IMAGE_VARIANT != "" ]] || die '${IMAGE_VARIANT} must be defined.'
  [[ $DOCKER_REPOSITORY != "" ]] || die '${DOCKER_REPOSITORY} must be defined.'
```